.github/workflows/main.yml

```
@@ -11,10 +11,9 @@ on:
   merge_group:
     branches:
       - main
-    
   workflow_call: # Allow to be called from the release workflow
   schedule:
-    - cron: '31 15 * * 0' # Run periodically to keep CodeQL database updated
+    - cron: '0 3 * * *' # Nightly run for performance validation
```

YAMLlint (1.37.1)

[warning] too few spaces before comment: expected 2

(comments)